bin/webpack.js

```diff
@@ -5,13 +5,48 @@
 	Author Tobias Koppers @sokra
 */
 var path = require("path");
-// Local version replace global one
-try {
-	var localWebpack = require.resolve(path.join(process.cwd(), "node_modules", "webpack", "bin", "webpack.js"));
-	if(__filename !== localWebpack) {
-		return require(localWebpack);
+
+function getNodeModulesPaths(startPath) {
+	var parts = startPath.split(path.sep);
+	var i = parts.length - 1;
+	var paths = [];
+	while(i >= 0) {
+		if(parts[i] === "node_modules") {
+			i--;
+			continue;
+		}
+		var dir = parts.slice(0, i).join(path.sep);
+		if(dir.length) {
+			paths.push(dir);
+		}
+		i--;
 	}
-} catch(e) {}
+	return paths;
+}
+
+function resolveLocalWebpack(startPath) {
+	var resolvedPath = __filename;
+	var paths = getNodeModulesPaths(startPath);
+	paths.some(function(dirPath, i) {
```

Comment:
@sokra Thanks! Done, but I used `var` because currently it's not fully refactored file for es6, later I could refactor it if needed (as I see by open PR about refactoring).